core-primitives/top-pool-author/src/author.rs

```diff
@@ -244,62 +282,46 @@ where
 		Ok(self.top_pool.ready(shard).map(|top| top.data().encode()).collect())
 	}
 
-	fn get_pending_tops_separated(
-		&self,
-		shard: ShardIdentifier,
-	) -> Result<(Vec<TrustedOperation>, Vec<TrustedGetterSigned>)> {
```

Comment:
I refactored this function into two separate functions for calls and getters. I left the getters function still in, in case we do want to have getters in the TOP pool again at some point in the future.

Reply:
This won't be the case. If we will ever have getters in a pool again, this will be a separate pool. (see our components discussion)

Reply:
Separate pool instance, yes - but maybe we re-use some of the authoring logic? But if you're adamant about removing all traces of Getters from the top pool author, I can do it..

Reply:
Hm. Nay, leave it for now. I think a bigger refactoring tasks will be necessary any way for the pool. Let's leave this for then.